spec/observables/forkJoin-spec.ts

```diff
@@ -11,13 +11,14 @@ declare const asDiagram: Function;
 describe('forkJoin', () => {
   asDiagram('forkJoin')
     ('should join the last values of the provided observables into an array', () => {
-      const e1 = forkJoin(
-                   hot('---a---b---c---d---|'),
-                   hot('-1---2---3---|')
-      );
-      const expected = '-------------------(x|)';
+      const e1 = forkJoin([
```

Comment:
please be sure the marble diagrams line up. Until we get our tests updated to use run mode, that means simply adding (gross looking) spaces to make sure the first single quotes align in the strings.

Reply:
Fixed